### PR TITLE
Chore: Configure ESLint

### PR DESCRIPTION
**Pull request by saszaCodes**
#### What does this PR do?
Adds ESLint to our project

#### Description of Task to be completed?
Configure ESLint

#### How should this be manually tested?
run `npm run lint` in the project directory, if no errors occur, everything works fine.

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
+
 const app = express();
 
-app.get('/', (req,res) => {
-  res.send("This is my first Express server");
+app.get('/', (req, res) => {
+  res.send('This is my first Express server');
 });
 
 app.listen(process.env.port || 3000);
```
Comment (Avuidrauxs) [inline src/app.js]: I will suggest you to do this ,

```suggestion
const PORT = process.env.port || 3000;

app.get('/', (req, res) => {
  res.send('This is my first Express server');
});

app.listen(PORT, () => {
  console.log('Listening on port ' + PORT);
});
```

**Review by Avuidrauxs — CHANGES_REQUESTED** [review]
Awesome job, just a few changes.

**Review by saszaCodes — COMMENTED** [review]
@Avuidrauxs ok, I implemented suggested changes. I resolved Hound's comments without addressing themm because I think it finds violations here only because hound.yml is not present in this branch, as I implemented it in the [next one](https://github.com/tepeszCodes/document-management-system/tree/chore/integrate-hound-ci).

**Review by Avuidrauxs — APPROVED** [review]
Looks good.

**Pull request merged**